src/components/EventDetail/index.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./EventDetail.css"
+import Header from "../Header/index";
+import Text from "../Text/index";
+
+const EventDetail = ({about, time, date}) => {
+    return (
+        <aside className="eventDetails">
```

Comment:
@kerry-okpere, using an `aside` here, this is good use of semantic HTML, really good!